src/obr/signac_wrapper/operations.py

```diff
@@ -21,11 +22,36 @@
 
 
 class OpenFOAMProject(flow.FlowProject):
+    filtered_jobs = None
+    queried_jobs = None
+
     def print_operations(self):
         ops = sorted(self.operations.keys())
         logging.info("Available operations are:\n\t" + "\n\t".join(ops))
         return
 
+    def get_jobs(self, filter=Union[None, dict[str, str]], queries=None):
+        self.filtered_jobs = self.filter_jobs(filters=filter)
+        resulting_jobs = self.query_jobs()
+        return resulting_jobs
+
+    def filter_jobs(self, filters: dict[str, str], output=True):
+        all_jobs: list[Job] = self.queried_jobs or self.find_jobs()
+        # chain filtering of jobs
+        for key, value in filters.items():
+            all_jobs = filter(
+                lambda job: value in self.get_job_status(job)[key], all_jobs
+            )
+        self.filtered_jobs = list(all_jobs)
```

Comment:
```suggestion
        return list(all_jobs)
```

And handle logging of what has been selected somewhere else?

Reply:
I would prefer to log as individual as possible to aid debugging and error handling. Where would you recommend moving the logging to? @greole

Reply:
Related to this #112  adds printing out the stack trace for failed operations.

Reply:
> I would prefer to log as individual as possible to aid debugging and error handling. Where would you recommend moving the logging to? @greole

Good point. I'll think about it.